packages/atomic/tsconfig.json

```diff
@@ -24,13 +24,16 @@
         }
       }
     ],
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
   },
   "include": ["src", "types/jsx.d.ts"],
   "exclude": [
     "node_modules",
     "src/external-builds",
     "**/*.stories.tsx",
     "headless"
-  ]
+  ],
+  "files": ["./node_modules/@applitools/eyes-cypress/index.d.ts"],
```

Comment:
Is this really needed ?

Reply:
Without this line, the linter flags the custom cypress commands like `cy.eyesCheckWindow`

Reply:
I hadn't noticed that there's a tsconfig file in the cypress folder. Moving this and the other changes there. Would that be OK?